### PR TITLE
Update INSTALL.md

### PR DESCRIPTION
**Pull request by nafur**
This PR updates our build and installation instructions in `INSTALL.md` after we have changed how we handle dependencies.
Thanks @polgreen / #6391.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -3,9 +3,10 @@ CVC4 prerelease version 1.9
 
 ## Building CVC4
 
-    ./contrib/get-antlr-3.4  # download and build ANTLR
-    ./configure.sh   # use --prefix to specify a prefix (default: /usr/local)
-                     # use --name=<PATH> for custom build directory
+    ./configure.sh
+        # use --prefix to specify an install prefix (default: /usr/local)
+        # use --name=<PATH> for custom build directory
+        # use --auto-download to download and build missing dependencies
```
Comment (HanielB) [inline INSTALL.md:9]:
```suggestion
        # use --auto-download to download and build missing required dependencies
```

File: INSTALL.md [hunk 2]
```diff
@@ -16,8 +17,7 @@ All binaries are built into `<build_dir>/bin`, the CVC4 library is built into
 
 ## Supported Operating Systems
 
-CVC4 can be built on Linux and macOS.  For Windows, CVC4 can be cross-compiled
-using Mingw-w64.  We recommend a 64-bit operating system.
+CVC4 can be built on Linux and macOS.  Cross-compilation is possible for Arm64 systems and Windows using Mingw-w64.  We recommend a 64-bit operating system.
```
Comment (HanielB) [inline INSTALL.md:20]: CVC5

File: INSTALL.md [hunk 3]
```diff
@@ -41,51 +41,31 @@ can be found in `<build_dir>/lib`.
 
 ## Build dependencies
 
-The following tools and libraries are required to build and run CVC4.
+CVC4 makes uses of a number of tools and libraries. Some of these are required while others are only used with certain configuration options.
```
Comment (HanielB) [inline INSTALL.md:44]: CVC5

File: INSTALL.md [hunk 4]
```diff
@@ -41,51 +41,31 @@ can be found in `<build_dir>/lib`.
...
 
-Some features, such as the theory of floating-point numbers, require
-[optional dependencies](optional-dependencies) (see below).
 
-### Installing libantlr3c: ANTLR parser generator C support library
+### ANTLR 3.4 parser generator
 
-For libantlr3c, you can use the script `contrib/get-antlr-3.4`.
-This will download, patch, and install libantlr3c.
+For most systems, the package manager no longer contains pre-packaged versions of ANTLR 3.4. With `--auto-download`, CVC4 will automatically download and build ANTLR 3.4.
```
Comment (HanielB) [inline INSTALL.md:61]: CVC5

**Review by HanielB — COMMENTED** [review]
As a general comment, maybe wrap lines with more than 80 chars?

File: INSTALL.md (hunk 1 above)
Comment (nafur) [reply INSTALL.md:9]: We also build missing dependencies that are optional, but enabled anyways.

Comment (HanielB) [reply INSTALL.md:9]: I see. Maybe "missing, required or enabled, dependencies"? I think it's somewhat redundant, but useful nonetheless.

**Comment by mpreiner** [comment]
Added do-no-merge since this should go in after #6371.

File: INSTALL.md [hunk 5]
```diff
@@ -337,34 +319,34 @@ available on the system. Override with `ARGS=-jN`.
...
-2. Determine the commit sha and configuration of the CVC4 binary
+2. Determine the commit sha and configuration of the CVC5 binary
 ```
-cvc4 --show-config
+cvc5 --show-config
 ```
 3. Download the specific source code version:
 ```
-wget https://github.com/CVC4/CVC4/archive/<commit-sha>.tar.gz
+wget https://github.com/CVC5/CVC5/archive/<commit-sha>.tar.gz
```
Comment (mpreiner) [inline INSTALL.md:337]: This repo also doesn't exist yet.

File: INSTALL.md [hunk 6]
```diff
@@ -1,33 +1,35 @@
...
 
-CVC4 can be built on Linux and macOS.  For Windows, CVC4 can be cross-compiled
-using Mingw-w64.  We recommend a 64-bit operating system.
+CVC5 can be built on Linux and macOS.  Cross-compilation is possible for Arm64
+systems and Windows using Mingw-w64.  We recommend a 64-bit operating system.
 
 On macOS, we recommend using Homebrew (https://brew.sh/) to install the
 dependencies.  We also have a Homebrew Tap available at
-https://github.com/CVC4/homebrew-cvc4 .
+https://github.com/CVC5/homebrew-cvc4 .
```
Comment (mpreiner) [inline INSTALL.md:26]: Repo does not exist yet.

File: INSTALL.md [hunk 7]
```diff
@@ -204,13 +186,13 @@ binding, please contact one of the project leaders.
 
 ## Building the API Documentation
 
-Building the API documentation of CVC4 requires the following dependencies:
+Building the API documentation of CVC5 requires the following dependencies:
 * [Doxygen](https://www.doxygen.nl)
 * [Sphinx](https://www.sphinx-doc.org)
 * [Breathe](https://breathe.readthedocs.io)
```
Comment (aniemetz) [inline INSTALL.md:192]: Add missing Python module dependencies:
* sphinx-tabs
* sphinxcontrib-bibtex

File: INSTALL.md [hunk 8]
```diff
@@ -1,33 +1,35 @@
-CVC4 prerelease version 1.9
+CVC5 prerelease version 1.9
```
Comment (aniemetz) [inline INSTALL.md:1]: Version should be 1.0, right?

**Review by aniemetz — CHANGES_REQUESTED** [review]
All occurrence of `CVC5` should be changed to `cvc5`.

**Review by aniemetz — APPROVED** [review]

**Pull request merged**